lume.yml

```diff
@@ -36,14 +36,18 @@ steps:
     envs:
       END2END_TEST: True
       GUNICORN_WORKERS: 1
-      TASK_REPOSITORY_TYPE: sqlite # declared already in pytest.ini
-      SQL_SERVER: sqlite # declared already in pytest.ini
-      SQL_DATABASE: taskmanager_test.db # declared already in pytest.ini
+      TASK_REPOSITORY_TYPE: mysql # declared already in pytest.ini
+      SQL_SERVER: mysql # declared already in pytest.ini
+      SQL_DATABASE: taskmanager
+      MYSQL_DATABASE: taskmanager
+      MYSQL_USER: root
+      MYSQL_PASSWORD: root
+      MYSQL_HOST: localhost
```

Comment:
This a bit weird. Since pytest creates an instance of petisco we need to set up the MySQL connection with the exposed port but on the docker container use the internal parameters.

Reply:
Fixed by avoiding petisco setup when running e2e tests.